pointgrey_camera_driver/cfg/PointGrey.cfg

```diff
@@ -76,14 +76,16 @@ gen = ParameterGenerator()
 
 video_modes = gen.enum([gen.const("Format0_Mode5", str_t, "640x480_mono8", ""),
                   gen.const("Format0_Mode6", str_t, "640x480_mono16", ""),
-		  gen.const("Format2_Mode1", str_t, "1280x960_bayer8", ""),
+		              gen.const("Format2_Mode1", str_t, "1280x960_bayer8", ""),
```

Comment:
Indentation looks wrong through this file, which makes it hard to edit (when things don't line up) and hard to read.